miden-crypto/benches/common/macros.rs

```diff
@@ -954,3 +954,81 @@ macro_rules! benchmark_word_conversions {
         }
     };
 }
+
+/// Generates comprehensive AEAD benchmarks for bytes operations.
+///
+/// This macro creates benchmarks for encryption and decryption operations
+/// using the new standardized approach with consistent data generation,
+/// throughput measurement, and reduced boilerplate.
+///
+/// # Arguments
+/// * `$aead_module` - The AEAD module name (e.g., aead_rpo)
+/// * `$group_prefix` - Human-readable prefix for benchmark group names
+///
+/// # Generated benchmarks
+/// - `benchmark_<module>_bytes` - Byte array encryption/decryption
+#[macro_export]
+macro_rules! benchmark_aead {
+    ($aead_module:ident, $group_prefix:expr) => {
+        paste::paste! {
```

Comment:
Paste is a relatively simple crate, but isn't well-maintained, hence used sparingly. It's typically reserved for places where you can't do otherwise, such as when you [*need* to programmatically generate identifiers](https://github.com/0xMiden/miden-vm/pull/2078/files#diff-28135d239794088c01ff4c474bb24bc42f577d831a79bcf418279d15514926a2R18-R24). Here I expect you need one name per macro call, which would make it easy to request this as a macro argument, no?